package.json

```diff
@@ -60,6 +60,7 @@
     "@react-navigation/drawer": "6.3.0",
     "@react-navigation/native": "6.0.11",
     "@react-navigation/stack": "6.2.2",
+    "@testing-library/react": "^13.4.0",
```

Comment:
I see you added a library here, can you commit the package-lock file too here?

Reply:
@Santhosh-Sellavel Pushed `package-lock` file too.